cmd/root.go

```diff
@@ -49,10 +63,21 @@ var rootCmd = &cobra.Command{
 			log.SetLevel(log.DebugLevel)
 			log.SetReportCaller(true)
 		}
+		// Configure Logrus
 		err = configureLogrusHooks()
 		if err != nil {
 			return errors.Wrap(err, "Unable to configure Logrus Hooks")
 		}
+		// Configure Honeycomb Telemetry
+		honeycombEnv, err := loadHoneycombEnv()
+		if err != nil {
+			return err
+		}
+		beeline.Init(beeline.Config{
+			WriteKey:    honeycombEnv.SECRET_KEY,
+			Dataset:     "aws-oidc",
+			ServiceName: "aws-oidc",
```

Comment:
Might also want to grab these, or at least the dataset, from the environment. And keep these values you have here as the default if env is not provided.